eventmesh-storage-plugin/eventmesh-storage-redis/src/main/java/org/apache/eventmesh/storage/redis/cloudevent/CloudEventCodec.java

```diff
@@ -26,7 +26,7 @@
 import io.netty.buffer.ByteBuf;
 import io.netty.buffer.ByteBufAllocator;
 
-
+// TODO: Duplicate with org.apache.eventmesh.connector.redis.cloudevent.CloudEventCodec
 public class CloudEventCodec extends BaseCodec {
```

Comment:
How about remove TODO, or replace with java doc

Reply:
It's done.